packages/@glimmer/syntax/test/loc-node-test.ts

```diff
@@ -134,6 +134,28 @@ test('html elements', () => {
   }
 });
 
+test('html elements with paths', () => {
+  let ast = parse(`
+    <Foo as |bar|>
+      <bar.x.y/>
+      <bar.x.y></bar.x.y>
+    </Foo>
+  `);
+
+  let [, section] = ast.body;
```

Comment:
what's the first node?

Reply:
Not sure, copied from tests above. Probably text node